### PR TITLE
For running parametric DE on all sample sizes added serverconfig.features.run_parametricDE

### PR DESCRIPTION
**Pull request by robinpaul85**
## Description
1) Added serverconfig.features.run_parametricDE to enable edgeR on all cases irrespective of sample sizes.
2) Other small changes so as to only parse the JSON string from edge.R script. This helps in printing out other output from the R script.

## Checklist

[Check each task](https://github.com/stjude/proteinpaint/wiki/Pull-Request-Checklist) that has been performed or verified to be not applicable.
- [ ] Tests: added and/or passed unit and integration tests, or N/A
- [ ] Todos: commented or documented, or N/A
- [ ] Notable Changes: updated release.txt, prefixed a commit message with "fix:" or "feat:", added to an internal tracking document, or N/A

### CODE REVIEW

File: server/routes/termdb.DE.ts [hunk 1]
```diff
@@ -98,8 +98,8 @@ param{}
 	const cases_string = group1names.map(i => i).join(',')
 	const controls_string = group2names.map(i => i).join(',')
 	const expression_input = {
-		case: cases_string,
-		control: controls_string,
+		case: controls_string,
```
Comment (xzhou82) [inline server/routes/termdb.DE.ts:101]: why swapped?

Comment (robinpaul85) [reply server/routes/termdb.DE.ts:101]: @karishma-gangwani wants the 1st group to be control and 2nd group to be treatment.

**Review by karishma-gangwani — APPROVED** [review]
looks good to me now. thanks @robinpaul85

File: server/routes/termdb.DE.ts (hunk 1 above)
Comment (karishma-gangwani) [reply server/routes/termdb.DE.ts:101]: later on we can let user choose. for now default control group is 1.

**Pull request merged**